easybuild/tools/module_generator.py

```diff
@@ -210,7 +210,7 @@ def get_description(self, conflict=True):
         """
         Generate a description.
         """
-        description = "%s - Homepage: %s" % (self.app.cfg['description'], self.app.cfg['homepage'])
+        description = "Description: %s - Homepage: %s" % (self.app.cfg['description'], self.app.cfg['homepage'])
 
```

Comment:
makes sense, good suggestion @geimer!

@pescobar: can you revert the change on this line, and change it below to:

```python
whatis = ["Description: %s" % description]
```